README.md

```diff
@@ -7,6 +7,20 @@ crowdsourcing workers, and from those judgments, to compute relevance metrics
 such as normalized discounted cumulative gain (NDCG) and mean average precision
 (MAP).
 
+## Running tests
+
+From a fresh arcs virtual environment, install pytest:
+
+```bash
+pip install pytest
```

Comment:
The tests will fail if you don't install the other modules necessary, and pytest is in the setup.py. Perhaps we should instead instruct to `pip install -e .`.